setup.py

```diff
@@ -18,7 +18,7 @@
     name="pytube",
     version=__version__,  # noqa: F821
     author="Nick Ficano, Harold Martin",
```

Comment:
Maybe change this to something like "Pytube organization", so we can avoid changing this as the project team evolves?

Reply:
sure, I'll keep this pr open until I get that setup